src/libraries/sendtohelixhelp.proj

```diff
@@ -201,6 +201,8 @@
         <AndroidInstrumentationName>net.dot.MonoRunner</AndroidInstrumentationName>
 
         <TestTimeout>$(_workItemTimeout)</TestTimeout>
+        <!-- The android sample returns 42 so it should be considered by xharness as a success -->
+        <ExpectedExitCode Condition="'%(Filename)' == 'HelloAndroid'">42</ExpectedExitCode>
```

Comment:
Why making Android sample return with this special number?

Reply:
The existing sample returns 42 https://github.com/dotnet/runtime/blob/master/src/mono/netcore/sample/Android/Program.cs#L11